### PR TITLE
Dispose baselines more carefully, especially workspace baselines.

### PR DESCRIPTION
**Pull request by merks**
Disable auto builds to reduce background activities.

### CODE REVIEW

File: apitools/org.eclipse.pde.api.tools.tests/src/org/eclipse/pde/api/tools/builder/tests/ApiTestingEnvironment.java [hunk 1]
```diff
@@ -507,6 +511,22 @@ protected IApiBaseline getWorkspaceProfile() {
 		return ApiPlugin.getDefault().getApiBaselineManager().getWorkspaceBaseline();
 	}
 
+	public static void dispose(IApiBaseline apiBaseline) {
+		if (apiBaseline instanceof WorkspaceBaseline) {
+			try {
+				IApiBaselineManager apiBaselineManager = ApiPlugin.getDefault().getApiBaselineManager();
+				Method disposeWorkspaceBaseline = ApiBaselineManager.class
+						.getDeclaredMethod("disposeWorkspaceBaseline"); //$NON-NLS-1$
```
Comment (iloveeclipse) [inline apitools/org.eclipse.pde.api.tools.tests/src/org/eclipse/pde/api/tools/builder/tests/ApiTestingEnvironment.java:519]: Why use reflection? Make the method public. I guess that's not a big deal, the manager is not API anyway

Comment (merks) [reply apitools/org.eclipse.pde.api.tools.tests/src/org/eclipse/pde/api/tools/builder/tests/ApiTestingEnvironment.java]: Done.

**Comment by merks** [comment]
It's a good chance to see if the build succeeds the a second time.

**Comment by merks** [comment]
@akurtakov @HannesWell @laeubi 

FYI, let's hope this also consistently fixes the problem for integration builds.

**Pull request merged**